dipper/sources/ZFIN.py

```diff
@@ -17,7 +19,7 @@
 
 LOG = logging.getLogger(__name__)
 ZFDL = 'http://zfin.org/downloads'
-
+OBOLIB = 'http://purl.obolibrary.org/obo/zp/src/curation'
```

Comment:
Variable name is misleading: Better use: **_ZPCURATION_**

Reply:
Agreed